src/arrays.js

```diff
@@ -1,49 +1,65 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length) 
+  return array[index - array.length];
+  return array[index]
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString()
 };
 
-const csvStringToArray = string => {
-  // your code here
-};
+function csvStringToArray(string) {
+  return string.split(",", 5)
+}
 
-const addToArray = (element, array) => {
-  // your code here
+const addToArray = (elementX, arrayX) => {
+  arrayX.push(elementX)
 };
 
-const addToArray2 = (element, array) => {
-  // your code here
+const addToArray2 = (NEWelement, ORIGarray) => {
+  let addToArray3;
+  addToArray3 = ORIGarray.concat(NEWelement)
+  return addToArray3
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+ return array.splice(index,1)
 };
 
+
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(String)
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(function(A){return A.toUpperCase();})
```

Comment:
When using `map`. you dont need to decalre an unnamed function like this, its created implicitly. If youre not sure about this it would be a good idea to have a look in the [docs for map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)
```
  return strings.map(string => string.toUpperCase());
```